package.json

```diff
@@ -23,7 +23,7 @@
     "build:dev": "yarn workspace @chakra-ui/vue run rollup --config rollup.dev.config.js",
     "build": "yarn workspace @chakra-ui/vue build",
     "dev": "yarn workspace @chakra-ui/vue dev",
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint packages/chakra-ui-core", 
```

Comment:
How does this affect the other packages? We should probably change this to:
`packages/*` and then in the `.eslintignore` file, we can uncomment the `chakra-ui-theme` line.

Reply:
let me check

Reply:
i think each package should have its own `eslintrc` file or lint script. Not sure if we should be using `vue-cli-service` in root package.json tho.

Reply:
Lets get this merged and later we should refactor eslint and package.json(root).

Reply:
I see. This is good for now. We can do some research on how to improve this later.